test/test_views.py

```diff
@@ -1,5 +1,5 @@
-from ThreeDiToolbox.views.graph import generate_parameter_config
-from ThreeDiToolbox.views.graph import parse_aggvarname
+from ThreeDiToolbox.utils.utils import generate_parameter_config
+from ThreeDiToolbox.utils.utils import parse_aggvarname
 
 import unittest
 
```

Comment:
@reinout deze generate_parameter_config en parse_aggvarname worden door zowel tool_graph en tool_animation gebruikt. Derhalve heb ik beide functie maar in utils gezet nu..

Reply:
Prima. Is er ook verkeer *naar* utils i.p.v. alleen eruit :-)